src/status_im/chat/events/input.cljs

```diff
@@ -476,18 +530,17 @@
               :dispatch [:update-text-selection new-sel]})))))))
 
 (register-handler-fx
- :select-next-argument
- (fn [{{:keys [chat-ui-props current-chat-id] :as db} :db} _]
-   (let [arg-pos (input-model/argument-position db current-chat-id)]
-     (let [input-text   (get-in db [:chats current-chat-id :input-text])
-           command-args (cond-> (input-model/split-command-args input-text)
-                          (input-model/text-ends-with-space? input-text) (conj ""))
-           new-sel      (->> command-args
-                             (take (+ 3 arg-pos))
-                             (input-model/join-command-args)
-                             count
-                             (min (count input-text)))
-           ref          (get-in chat-ui-props [current-chat-id :input-ref])]
-       {::set-native-props {:ref ref
-                            :props {:selection {:start new-sel :end new-sel}}}
-        :dispatch [:update-text-selection new-sel]}))))
+ :set-contact-as-command-argument
+ [trim-v]
+ (fn [{:keys [db]} [bot-db-key contact arg-index name]]
```

Comment:
Can we turn the big vector arg into a map? 4 positional args is pushing it in terms of not accidentally calling it wrong.